src/api/ums/umsModule.spec.ts

```diff
@@ -24,13 +24,19 @@ describe("UMS Module", () => {
     projectId = faker.random.uuid(),
     access_token = createTestToken(),
     refresh_token = faker.random.uuid(),
+    email = faker.internet.email(),
     user = {
-      email: faker.internet.email(),
+      email,
```

Comment:
not `email: email,`?
Or is this typescript magic?

Reply:
Its a real object with values here and not a `type` 😉

Reply:
`email` is set using `faker.internet.email()`, so I guess it is a string containing a (fake) email address?
So before you used
```
user = {
  email: faker.internet.email(),
// etc
}
```

now you replaced with
```
user = {
  email, // where `email` is a string?
// etc
}
```

So I wondered why this would be the same (just out of interest for me personally I guess, for when I have to become a FE dev :wink: )

Reply:
I see 😄 , well that is ES6 magic. 

You can set values for an object by:

```
const email = 'john@doe.com';
const object = {
  email: email,
}
```

Or a bit shorter, by omitting the value.

```
const email = 'john@doe.com';
const object = {
  email,
}
```

For more info see --> https://alligator.io/js/object-property-shorthand-es6/

As you notice I set the email as a different property just above and then assign in back in this object.

Reply:
Ah... so it is (standard) ES6 magic and not TypeScript :smile: 
I was guessing this would be the case, but guessed slightly wrong.
Learned something new!